charts/jaeger/templates/ingester-deploy.yaml

```diff
@@ -0,0 +1,149 @@
+{{- if .Values.ingester.enabled -}}
```

Comment:
Definitely something we want to do! But Please open a separate PR where we tackle just this.